docs/topics/proc_exposing-application-metrics-using-prometheus-on-node-js.adoc

```diff
@@ -0,0 +1,134 @@
+[id='exposing-application-metrics-using-prometheus-on-node-js']
+= Exposing application metrics using Prometheus on {Node}
+
+In this example, you:
+
+* Configure your application to expose metrics.
+* Collect and view the data using Prometheus.
+
+Note that Prometheus actively connects to a monitored application to collect data; the application does not actively send metrics to a server.
+
+.Prerequisites
+
+* Prometheus configured to collect metrics from the application:
++
+--
+. Get the default Prometheus configuration using the following command:
++
+[source,bash,opts="nowrap"]
+----
+# docker run -it --rm --entrypoint /bin/sh prom/prometheus -c 'cat /etc/prometheus/prometheus.yml'
+----
++
+Store the output in the `prometheus.yml` file in the directory with your application.
+
+. Append the following snippet to the `prometheus.yml` file to make Prometheus automatically collect metrics from your application:
++
+[source,yaml]
+----
+  - job_name: nodejs
+     static_configs:
+     - targets: ['localhost:8888']
+----
++
+The default behavior of {Node}-based applications is to expose metrics at the `/metrics` endpoint.
+This is also what Prometheus expects.
+--
+
+* The Prometheus server running on `localhost` using Docker:
++
+--
+[source,bash,opts="nowrap"]
+----
+# docker run -it --rm --network host -p 9090:9090 -v $PWD/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus
+----
+
+Wait until the `Server is ready to receive web requests` message is displayed in the console.
+
+When running your application in a Linux environment, include the the `--network host` option to allow Prometheus to connect to your application.
+
+NOTE: The `--network host` option is available only in Linux environments.
+It is not supported on Windows or MacOS. 
```

Comment:
There needs to be _some_ way for Prometheus to connect to the application. Using `--network host` is easiest, and actually I think the _only_ possible way if we keep the application running on `localhost`. Host networking is also used in the Thorntail variant of this guide.

If we care enough about Windows and Mac, we have to rework the entire guide to some extent.

Merely saying that host networking only works on Linux and that you should only add `--network host` when on Linux doesn't work.

Reply:
@Ladicek thank you for the clarification:)
I think for this delivery, the guide should focus on running the application in a Linux environment.
We haven't defined a requirement for this procedure to be applicable to Windows and MacOS environments, and rather than delivering a half-completed multi-environment workflow. I'd focus on delivering one that is complete and consistent for all runtimes on Linux.
Also, we are likely going to rework this procedure extensively for the next train, as we plan to make it work with an OpenShift cluster rather than localhost.

Reply:
Ayea. I'd just revert to the previous wording.